Fixed reading of configuration ack

saarnold opened the pull request:
- avoid to read a configuration ack after start acquisition 
 After the acquisition was started, using the CS command, the device leaves the
configuration mode. Therefor no further configuration ack will be send.
My guess why this was working most of the time before is that there might have been
enough ack's in the buffer due to the carriage return's send in setConfigurationMode
- use read timeout property when reading the configuration ack

saarnold committed:
avoid to read a configuration ack after start acquisition

After the acquisition was started, using the CS command, the device leaves the
configuration mode. Therefor no further configuration ack will be send.
My guess why this was working most of the time before is that there might have been
enough ack's in the buffer due to the carriage return's send in setConfigurationMode

doudou commented:
Mostly looks good,

> My guess why this was working most of the time before is that there might have been
enough ack's in the buffer due to the carriage return's send in setConfigurationMode

If that's the case, shouldn't you increase the timeout there, to be sure that no extra carriage returns will be received ?

saarnold committed:
use read timeout property when reading the configuration ack

Initialize read timeout with 1 second in order to align with the current default value

saarnold commented:
> If that's the case, shouldn't you increase the timeout there, to be sure that no extra carriage returns will be received ?

The origin of the extra carriage returns was indeed in the setConfigurationMode method. It uses now also the read timeout property. Thanks!
